tests/common.go

```diff
@@ -139,7 +141,12 @@ func InitInstance() {
 		token = ""
 	}
 
+	// Set driver namespace
+	driverNamespace, err := schedops.GetDriverNamespace()
```

Comment:
change it to volumeDriverNamespace

Reply:
We really need to all sync up and come up with a name we all agree on, keep changing the name is waste of time.

Reply:
Done, renamed all `driverNamespace` to `volumeDriverNamespace` as per our talk in the morning sync up.